### PR TITLE
Multi-arch build for amd64, arm64, armv7 with buildx on Cloud Build

### LINKED ISSUE #62: Add CI configuration that would build ready-to-use containers for all major platforms
Add CI configuration ([GithubActions](https://github.com/features/actions) or [CloudBuild](https://cloud.google.com/cloud-build)) that would build ready-to-use containers for all major platforms (e.g. linux/amd64, linux/arm, linux/arm64)

- [x] Tag images with:
  - full version (e.g. 1.0.2)
  - minor version (e.g. 1.0)
  - major version (e.g. 1)
  - latest git ref (e.g. `git rev-parse --short HEAD`)
  - Build ID for debugging/tracking purposes

- [ ] Cross compile images for `linux/amd64, linux/arm, linux/arm64`

### PR DESCRIPTION
**Pull request by SoapHia**
Modified the cloud build configuration to build for `arm64`,`amd64`, `armv7` architectures. 

YAML linter re-formatted the entire file, but the only lines changed are 26-54, 98, 101-102. These changes were based on an [example from GCP](https://github.com/GoogleCloudPlatform/solutions-build-multi-architecture-images-tutorial/blob/master/terraform/cloud-build/build-docker-image-trigger.yaml). 

The new build will produce a docker manifest list which contains 3 images (one per architecture). It uses a combination of [docker buildx](https://docs.docker.com/buildx/working-with-buildx/#high-level-build-options) and [QEMU](https://github.com/multiarch/qemu-user-static) to build images cross platform. 

Closes #62

### CODE REVIEW

File: ci/cloudbuild.yaml [hunk 1]
```diff
@@ -1,84 +1,105 @@
...
-# errors from docker pull, not just when image doesn't exist. Consider using
-# a more optimal approach which can parse the output from docker pull to handle
-# errors better.
-- name: 'gcr.io/cloud-builders/docker'
-  entrypoint: '/bin/bash'
-  args: ['-c', 'docker pull gcr.io/$PROJECT_ID/ts-bridge:latest || exit 0']
+  - name: "gcr.io/cloud-builders/docker"
+    id: "initialize-qemu"
+    args:
+      ["run", "--privileged", "multiarch/qemu-user-static", "--reset", "-p yes"]
```
Comment (Temikus) [inline ci/cloudbuild.yaml:29]: Can you specify what `-p yes` and `--reset` are needed for?

File: ci/cloudbuild.yaml [hunk 2]
```diff
@@ -1,84 +1,105 @@
...
+  - name: "docker.io/aquasec/trivy:latest"
+    id: "Scan newly built image using Trivy and create trivy-out.table"
+    args:
+      - "image"
+      - "--output=trivy-out.table"
+      - "--no-progress"
+      - "gcr.io/cre-tools/ts-bridge:git-$SHORT_SHA"
+
+  - name: gcr.io/cloud-builders/gcloud
+    entrypoint: "bash"
```
Comment (Temikus) [inline ci/cloudbuild.yaml:74]: Can you add a comment on what this action does?

File: ci/cloudbuild.yaml [hunk 3]
```diff
@@ -1,84 +1,105 @@
...
+          --repo_name=$_REPO_NAME \
+          --token_file=git_token.txt \
+          --trivy_file=trivy-out
 substitutions:
   _REPO_NAME: google/ts-bridge
+  _DOCKER_BUILDX_PLATFORMS: "linux/arm64,linux/amd64,linux/arm/v7"
 options:
-  machineType: 'N1_HIGHCPU_8'
+  machineType: "N1_HIGHCPU_8"
+  env:
```
Comment (Temikus) [inline ci/cloudbuild.yaml:101]: Can you add a comment on why this is needed and whether we're planning to remove it in the future?

**Review by Temikus — APPROVED** [review]
VERY good job, thank you! Just some minor nits left.

File: ci/cloudbuild.yaml (hunk 2 above)
Comment (SoapHia) [reply ci/cloudbuild.yaml:83]: Done :)

File: ci/cloudbuild.yaml (hunk 3 above)
Comment (SoapHia) [reply ci/cloudbuild.yaml:110]: Done :)

File: ci/cloudbuild.yaml (hunk 1 above)
Comment (SoapHia) [reply ci/cloudbuild.yaml:33]: Done :)

**Review by Temikus — APPROVED** [review]

**Pull request merged**